packages/components/sidebar/test/sidebar-item.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { mount } from '@vue/test-utils'
+import { describe, it, expect } from 'vitest'
+import PuikSidebarItem from '../src/sidebar-item.vue'
+import type { MountingOptions, VueWrapper } from '@vue/test-utils'
+
+let wrapper: VueWrapper<any>
+const factory = (template: string, options: MountingOptions<any> = {}) => {
+  wrapper = mount({
+    components: {
+      PuikSidebarItem,
+    },
+    template,
+    ...options,
+  })
+}
+
+describe('Sidebar tests', () => {
+  it('should be a vue instance', () => {
+    const template = `
+      <puik-sidebar-item title="title" />
+    `
+    factory(template)
+    expect(wrapper).toBeTruthy()
+  })
+
+  it('should be active', () => {
```

Comment:
Il manque des tests sur le rendu conditionnel des boutons via la props href/to, sur le titre et sur l'icon

Reply:
Je me suis demandé si le test du rendu conditionnel revenait pas a tester le bouton, c'est pour ca que je l'ai pas inclu